pkg/messaging/grpc_proxy.go

```diff
@@ -98,6 +98,10 @@ func (p *proxy) intercept(ctx context.Context, fullName string) (context.Context
 		return outCtx, conn, cErr
 	}
 
+	if p.remoteAppFn == nil {
```

Comment:
Please move the if block before line 83: https://github.com/dapr/dapr/blob/a782b85439c7d3ed87a76b2eb086e2f9eedb89b4/pkg/messaging/grpc_proxy.go#L83.

Reply:
Oh, bad rebase from release-1.5 to master. Good catch.